tools/wptrunner/wptrunner/executors/executorcontentshell.py

```diff
@@ -91,7 +91,9 @@ def _read_text_block(self, deadline=None):
         while True:
             line = _read_line(self.stdout_queue, deadline, "utf-8", "replace", False)
 
-            if line.endswith(self.eof_marker):
+            if line.startswith('#LEAK'):
+                raise CrashError()
```

Comment:
Why we should raise CrashError?

Reply:
OK, it might be ok to use Crash to denote a leak. Should we move this to line 22?

Reply:
I'll run a test with it on line 22, I think it should be fine actually to look in both queues since if there is a "#LEAK" it can happen at any time and we definitely want to raise the error.